### PR TITLE
updated project name in README #4539

### LINKED ISSUE #4539: Update README
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-orga-server

Update the README.md file for the new Open Event API.
- Update the name of the repo in readme

### PR DESCRIPTION
**Pull request by Anonymous26**
Fixes #4539 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `nextgen` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by Anonymous26** [comment]
Sorry for delay but the work is done

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The Open Event API Server can be easily deployed on a variety of platforms. Deta
 
 One-click Heroku deployment is also available:
 
-[![Deploy to Docker Cloud](https://files.cloud.docker.com/images/deploy-to-dockercloud.svg)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-orga-server)  [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+[![Deploy to Docker Cloud](https://files.cloud.docker.com/images/deploy-to-dockercloud.svg)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-api-server)  [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (invinciblycool) [inline README.md]: Please check that the name of the repo has changed to `open-event-server`

File: README.md [hunk 2]
```diff
@@ -247,7 +247,7 @@ We have the following branches :
  * **master**
    This contains shipped code. After significant features/bug-fixes are accumulated on development, we make a version update, and make a release. (https://eventyay.com - Is running off the master branch. (whichever is the latest release.) Hosted on Google Cloud Platform (Google Container Engine + Kubernetes).)
  * **nextgen**
-	 All development for the `nextgen` version of the Open Event Orga Server (aka Open Event API Server) goes on in this branch. If you're making a contribution, please make a pull request to _nextgen_.
+	 All development for the `nextgen` version of the Open Event Orga Server (aka Open Event Server) goes on in this branch. If you're making a contribution, please make a pull request to _nextgen_.
```
Comment (invinciblycool) [inline README.md]: Also please change `nextgen` to `development`

**Comment by Anonymous26** [comment]
updated the required changes @invinciblycool  please review

**Comment by Anonymous26** [comment]
all good or any other change required @poush @SaptakS

**Comment by SaptakS** [comment]
Please resolve the merge conflicts.

**Comment by Anonymous26** [comment]
conflicts merged @SaptakS

**Comment by poush** [comment]
@invinciblycool Please update your review here.

**Pull request merged**